tests/framework/config/client.go

```diff
@@ -41,3 +42,8 @@ type CompactOption struct {
 	Physical bool
 	Timeout  time.Duration
 }
+
+type DefragOption struct {
+	Timeout time.Duration
+	WithUtl bool
```

Comment:
Let's not add `WithUtl` to the common framework, as it currently cannot be implemented for integration tests.